webapi/src/main/twirl/queries/sparql/v1/searchExtendedGraphDB.scala.txt

```diff
@@ -465,7 +459,7 @@ WHERE {
                 }
 
                 @if(searchCriterion.matchBooleanNegativeTerms.nonEmpty) {
-                    MINUS {
+                    FILTER NOT EXISTS {
```

Comment:
Is this intended? I remember that we had some issues with unexpected behaviour with `FILTER NOT EXISTS` and `MINUS`.